### DISCUSSION TITLE
laravel/framework replaces illuminate/support and thus cannot coexist with it

### DISCUSSION THREAD

**Discussion by pravin298** [laravel/nova-issues#5451] [Q&A]
Below error while doing composer install:
Only one of these can be installed: illuminate/support[v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev], laravel/framework[v6.0.0, ..., 6.x-dev]. laravel/framework replaces illuminate/support and thus cannot coexist with it.

I am trying to upgrade laravel 5.8 to 6.0.

**Comment by crynobone** [comment]
Please provide the full error instead of a snippet.
